cmd/motion/main.go

```diff
@@ -76,6 +91,30 @@ func main() {
 					return err
 				}
 				store = rbstore
+			} else if cctx.Bool("experimentalSingularityStore") {
+				singularityAPIUrl := cctx.String("remoteSingularityAPIUrl")
+				var client client.Client
+				if singularityAPIUrl != "" {
+					client = httpclient.NewHTTPClient(http.DefaultClient, singularityAPIUrl)
+				} else {
+					db, err := database.OpenWithDefaults("sqlite:" + storeDir + "/singularity.db")
```

Comment:
I am curious what would the benefit of supporting this? 

For the Singularity options to work, singularity needs to be running anyway and running singularity API to then only communicate remotely seems like a that that is much less error prone and would give Singularity full ownership of its own database.